projects/cashmere/src/lib/button/button.component.scss

```diff
@@ -82,7 +82,7 @@ $btn-icon-sz: 18px;
     &.hc-secondary {
         @include colorButton($gray-100, 5%);
         color: $gray-600;
-        border: 2px solid $gray-300;
+        border: 1.5px solid $gray-300;
         height: 37px;
```

Comment:
Thanks for updating this style as we discussed to match the stroke on the select box.  One thing to change though is to make `height: 36px` instead of 37px now.  You can check the example on the subnav component to see a primary and secondary button side by side.